sample-submodule-2/src/main/java/com/airbnb/android/submodule/showkasesample/LocalePreview.kt

```diff
@@ -0,0 +1,6 @@
+package com.airbnb.android.submodule.two.showkasesample
+
+import androidx.compose.ui.tooling.preview.Preview
+
+@Preview(name = "English", group = "LocalePreview", locale = "en")
+annotation class LocalePreview
```

Comment:
Remove any custom preview annotations from this module. Instead, make this module have a dependency on `sample-submodule`.

Have a preview in `sample-submodule2` that is annotated by one of the custom annotations declared in a different module. I want to see how that setup would behave. Test it with both ksp and kapt.

Reply:
Added a case for this now, and it works with KAPT if the compiler argument with the qualifierd name is provided :)